### PR TITLE
Healing wand/staff regenerate limbs on use

### PR DESCRIPTION
**Pull request by Incoming5643**
if your missing a limb you'll grow it back, but if you have something else in the slot it won't be reverted to what it usually is.

Shit's magic.

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -768,6 +768,8 @@
 
 		for(var/datum/disease/D in viruses)
 			D.cure(0)
+			
+	regenerate_limbs()
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon.dm]: This is done on `/human` if `admin_revive` is 1.

Comment (Incoming5643) [inline code/modules/mob/living/carbon/carbon.dm]: Alright well I still want wands/staves of healing to fix this so I guess I'll just throw that bit in the bullet

Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon.dm]: for some reason `admin_revive` removes cuffs though so uh

Comment (phil235) [inline code/modules/mob/living/carbon/carbon.dm]: yeah you can't use admin_revive for things other than admin reviving, it removes cuffs, regrow limbs and plenty of other stuff.

**Pull request merged**